app/customer.py

```diff
@@ -0,0 +1,47 @@
+from math import sqrt
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self,
+                 name: str,
+                 product_cart: dict,
+                 location: list[int],
+                 money: float,
+                 car: dict) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
```

Comment:
The `location` attribute should be a tuple instead of a list to better represent a fixed pair of coordinates.